rs/node_rewards/canister/proto/rewards_calculator/pb/v1/types.proto

```diff
@@ -5,6 +5,10 @@ package rewards_calculator.pb.v1;
 import "ic_base_types/pb/v1/types.proto";
 import "ic_nervous_system/pb/v1/nervous_system.proto";
 
+enum RewardsCalculatorVersion {
+  REWARDS_CALCULATOR_VERSION_V1 = 0;
```

Comment:
You need to change this to be 
REWARDS_CALCULATOR_VERSION_UNSPECIFIED = 0;
REWARDS_CALCULATOR_VERSION_V1 = 1;

We always need an unspecified.